library/src/main/java/com/bumptech/glide/load/engine/cache/MemorySizeCalculator.java

```diff
@@ -129,7 +129,6 @@ private static boolean isLowMemoryDevice(ActivityManager activityManager) {
     static final int BITMAP_POOL_TARGET_SCREENS = 4;
     static final float MAX_SIZE_MULTIPLIER = 0.4f;
     static final float LOW_MEMORY_MAX_SIZE_MULTIPLIER = 0.33f;
-    // 4MB.
```

Comment:
Retain the comment? Or reference the constant in ArrayPool and remove the one here?